sql/parser/builtins.go

```diff
@@ -549,7 +549,19 @@ var builtins = map[string][]builtin{
 		}),
 	},
 
-	// TODO(thschroeter): implement round
+	"round": {
+		builtin{
+			returnType: DummyFloat,
+			types:      typeList{floatType, intType},
```

Comment:
You should also add `round(<float>)` which I believe behaves the same as `round(<float>, 0)`.